mesures/autoconsumo.py

```diff
@@ -57,7 +57,8 @@ def reader(self, file_path):
             df['emmagatzematge'] = np.where(df['emmagatzematge'], 'S', 'N')
             #df['potencia_nominal'] = np.where(df['cil'], '', df['potencia_nominal'])
             df['subgrup'] = df['subgrup'].apply(lambda x: x.replace('.', '')[:2])
-            df['tipus_antiabocament'] = df['tipus_antiabocament'].apply(lambda x: x.zfill(2))
+            df['tipus_antiabocament'] = (
+                df['tipus_antiabocament'].apply(lambda x: '' if (x == 0 or x is None or x == '') else x.zfill(2)))
```

Comment:
Utilitzant `not x` validem `False` i `None`
```suggestion
                df['tipus_antiabocament'].apply(lambda x: '' if (x == 0 or not x or x == '') else x.zfill(2)))
```